### ISSUE TITLE
WCAG check updaten naar 2.2

### ISSUE THREAD

**Issue by NielsHoffmann** [Geonovum/NL-ReSpec-template#12]
Zie mijn vraag op slack/durftevragen.

Huidige check is WCAG 2.0, volgens mij willen we conform overheidsrichtlijnen op 2.2 testen?
